src/Compilers/CSharp/Test/Semantic/Semantics/CollectionLiteralTests.cs

```diff
@@ -4606,7 +4718,6 @@ static string[] Append(string a, string b, bool c)
                     }
                 }
                 """;
-            // PROTOTYPE: Should spread elements support target type? (Should we infer IEnumerable<string>?)
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=15&length=9)

Added open issue to test plan [https://github.com/dotnet/roslyn/issues/66418](https://github.com/dotnet/roslyn/issues/66418).